recipes/strip-hints/meta.yaml

```diff
@@ -0,0 +1,78 @@
+  sha256: 828dcfe5104296229171ade81ceeefe81d780490d69ab779168704650016db3b
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - strip-hints = strip_hints.strip_hints_main:process_command_line
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - strip_hints
+
+about:
+  home: https://github.com/simplejson/simplejson
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: MIT
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: MIT
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE.txt
+  summary: 'Function and command-line program to strip Python type hints.'
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    This package provides a command-line command and a corresponding importable function that strips type hints
+    from Python code files. The stripping process leaves runnable code, assuming the rest of the code is runnable
+     in the interpreter version. The program tries to make as few changes as possible to the processed code so that
+      line and column numbers in error messages for the processed code file also correspond to those for
+      the original code file. In most cases, with the default options, both the line and column numbers are preserved.
+  dev_url: https://github.com/abarker/strip-hints
+
+extra:
+  recipe-maintainers:
+    - tanguycdls
+    - radcheb
```

Comment:
@radcheb are you willing to be a maintainer?

Reply:
Ok to become a maintainer of `strip-hints` conda package.